src/app/clusters/groups-server/groups-server.cpp

```diff
@@ -230,95 +277,64 @@ bool emberAfGroupsClusterViewGroupCallback(EndpointId endpoint, app::CommandHand
     return true;
 }
 
-bool emberAfGroupsClusterGetGroupMembershipCallback(EndpointId endpoint, app::CommandHandler * commandObj, uint8_t groupCount,
-                                                    uint8_t * groupList)
+/**
+ * @brief  Cluster GetGroupMembership Command callback (from client)
+ *
+ * WARNING: This function is UNTESTED. The groupList parameter is an array of structures,
+ * which is currently not supported. In consequence, the groupList parameter fails to decode,
+ * and this function is not being called.
```

Comment:
Is this because we do not yet have list/structs support in IM?

Reply:
That is correct. As far as I know, we can't send a command with struct list typed argument.